guidelines.md

```diff
@@ -1,15 +1,15 @@
 # Miruku.cafe guidelines
 
-1. Do not spam. Spamming is the repeated uploading of the same or similar content over and over for where there is no real effort put into the content.
+1. Do not spam. Spamming is the repeated uploading of the same or similar content over and over for where there is no real effort put into the content
 
 2. Do not post NSFW or suggestive content with minors or characters of questionable age, including drawn or otherwise fictional content
 
-3. Do not participate in hate speech, violent extremism
+3. Do not participate in hate speech or violent extremism
```

Comment:
```suggestion
3. Do not participate in hate speech. This includes violent extremism
```

Reply:
Violent extremism and hate speech are intertwined, but they are seperate things. (this is basically what @user51112 said)